frontend/src/services/auth.service.js

```diff
@@ -1,5 +1,6 @@
 import axios from "axios";
 
+//I don't really understand line 4../api/auth/ why?
```

Comment:
There's no real reason. It's just because we called it `/api/auth` on the backend and it looks cool